index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <div class="container">
+        <header class="header">
+            <h1 id="title">
+              Albert & Newton Survey Form
+            </h1>
+            <p id="description" class="description">Kindly take time to fill our survey form</p>
+        </header>
+        <form id="survey-form">
+            <div class="form-group">
+              <label for="name" id="name-label">Name</label>
+              <input type="text" class="form-control" name="name" id="name" placeholder="Enter your name" required>
+            </div>
+            <div class="form-group">
+              <label for="email" id="email-label">Email</label>
+              <input type="email" class="form-control" name="email" id="email" placeholder="Enter your email" required>
+            </div>
+            <div class="form-group">
+              <label for="number" id="number-label">Age<span class="hint">(optional)</span></label>
+              <input type="number" class="form-control" name="age" id="number" placeholder="Your Age" min="14" max="99" required>
+            </div>
+            <div class="form-group">
+                <p>Which language do you program in?</p>
+                <select name="role" id="dropdown" class="form-control" required>
+                  <option disabled selected value>Select an option</option>
+                  <option value="python">Python</option>
+                  <option value="ruby">Ruby</option>
+                  <option value="javascript">Javascript</option>
+                  <option value="php">PHP</option>
+                </select>
+            </div>
+            <div class="form-group">
+                <p>Would you advice your friend to start programming?</p>
+                <label>
+                  <input type="radio" name="user-recommend" value="definitely" class="input-radio" checked />Definitely
+                </label>
+                <label>
+                  <input type="radio" name="user-recommend" value="maybe" class="input-radio"/>Maybe
+                </label>
+                <label>
+                  <input type="radio" name="user-recommend" value="not-at-all" class="input-radio"/>Not at all
+                </label>
+            </div>
+            <!-- <div class="form-group">
+                <p>Which IT field do you love working on?</p>
+                <select name="mostLike" id="most-like" class="form-control" required>
+                  <option disabled selected value>Select an option</option>
+                  <option value="webdeveloper">Frontend</option>
+                  <option value="backend">Backend</option>
+                </select>
+            </div> -->
```

Comment:
While comments can be useful in explaining **complex logic or documentations**, it is essential to have **clean code** and consider the type of comment you have in your code. I think the above comment is _cluttering your code_ and _making it dirty_. I would like to recommend you to clean it up and make sure your code looks clean and professional :sunglasses::business_suit_levitating: